pymc3/distributions/transforms.py

```diff
@@ -238,6 +242,33 @@ def jacobian_det(self, x):
 upperbound = UpperBound
 
 
+class Ordered(Transform):
+    name = "ordered"
+
+    def backward(self, y):
+        x = tt.zeros(y.shape)
+        x = tt.inc_subtensor(x[..., 0], y[..., 0])
```

Comment:
This looks trivial to have figured out :P.

Reply:
I know - it blows my mind when I came across this.